api/meta.json

```diff
@@ -0,0 +1 @@
+{"title": "hello"}
```

Comment:
Remove this. Something in the tests is creating this file, I think it's my fault.